http-disk-server

```diff
@@ -353,7 +428,8 @@ def run_server(disk, ip, port):
             eprint('Unhandled exception: `{}`.'.format(e))
             eprint(traceback.format_exc())
             eprint('Restarting server...')
-            time.sleep(1)
+            if not STARTING_SERVER:
+                time.sleep(1)
```

Comment:
Is there a risk of looping very fast and consuming too much resources if there's something very wrong in the server initialization above?